src/components/lab/ListItemView/ListItemView.scss

```diff
@@ -0,0 +1,285 @@
+@use '../../variables';
+@use '../../../../styles/mixins';
+
+$block: '.#{variables.$ns}lab-list-item-view';
+
+/* ListItemView CSS API */
+#{$block} {
+    /* Sizes */
+    --_--g-list-item-view-height: var(--g-list-item-view-height, #{variables.$m-height});
```

Comment:
For "private" vars we don't need to specify the full name, `--_--height` will be enough

Reply:
Not sure about this, it may accidentally set the value to another component.